lib/debezium/types.go

```diff
@@ -95,33 +95,56 @@ func toInt64(value any) (int64, error) {
 	return 0, fmt.Errorf("failed to cast value '%v' with type '%T' to int64", value, value)
 }
 
+func (f Field) shouldCastToInt64() bool {
+	if f.IsInteger() {
+		return true
+	}
+
+	switch f.DebeziumType {
+	case
+		Timestamp,
```

Comment:
These are all of the types from https://github.com/artie-labs/transfer/pull/441/files#diff-6f8a144d4abeb92f8240e55383e15b8323e83bb3b9f69088274ed61f4f344e96R169